vehicle_gateway_betaflight/src/vehicle_gateway_betaflight.cpp

```diff
@@ -242,11 +242,20 @@ vehicle_gateway::VEHICLE_TYPE VehicleGatewayBetaflight::get_vehicle_type()
   return vehicle_gateway::VEHICLE_TYPE::ROTARY_WING;
 }
 
-vehicle_gateway::ARM_DISARM_REASON VehicleGatewayBetaflight::get_arm_reason() {}
+vehicle_gateway::ARM_DISARM_REASON VehicleGatewayBetaflight::get_arm_reason()
+{
+  return vehicle_gateway::ARM_DISARM_REASON_NONE;
```

Comment:
We should add `TODO`s in these sections since this is a temporary fix to get betaflight compiling

Reply:
Oh yeah, good catch